content/2017-09-19_pycon-cz-conference-wrap-up-summary.md

```diff
@@ -0,0 +1,199 @@
+
+Workshops aimed towards beginners ([testing](https://cz.pycon.org/2017/speakers/detail/workshop/7/), [robots](https://cz.pycon.org/2017/speakers/detail/workshop/1/), [programming for kids](https://cz.pycon.org/2017/speakers/detail/workshop/4/), Django Girls) significantly raised the numbers of women as well.
+
+In past years the Czech Python community made a big leap towards attracting women into tech. We think the attendance of women at PyCon CZ is significantly influenced by the day-to-day work of all who put the effort, primarily all people around the activites of [PyLadies](http://pyladies.cz/). We hope to become an inspiration for other Czech tech communities and events.
+
+### What Could Have Been Done
+
+- Given the context we're a developer-focused conference in the Czech Republic, ⅕ of audience a good number, but in the following years we would like to find and remove as much remaining obstacles for women to attend as possible and to achieve even higher numbers. Experience from abroad shows there's still a margin.
+- [Because of last minute changes of the venue](http://blog.python.cz/pycon-cz-2017-changes-date-and-venue), both of the main conference days happened during working days. According to reports, such choice isn't exactly the best for attracting beginners.
+- For the same reason the Django Girls workshop had to be on Sunday, after the whole conference. That's why many attendees of the workshop did not come to the conference, although they were given a free ticket.
+- We did not manage to pursue more partnerships. E.g. last year we invited [VODICÍ PES](www.vycvikvodicichpsu.cz).
+
+## Code of Conduct Report
+
+The organization team agrees that all PyCon events should be accessible and welcoming to everyone, regardless of their background. We agree that a clear and specific Code of Conduct is a necessity for any event.
+
+- The [Code of Conduct](https://cz.pycon.org/2017/about/code/) was based on Conference anti-harassment policy from [Geek feminism wiki](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy) and edited to be as specific to our event as possible
+- Agreeing to the Code was mandatory when buying a ticket
+- Two contact persons (male and female) were assigned
+- Attendees were reminded of the Code in the opening speech of each day
+
+### What Could Have Been Done
+
+- Summary of the Code in both languages were not posted around the venue
+- Phone numbers of the contact persons were not posted around the venue
+- There was no easy way to report an incident other than to write an e-mail or to talk to the CoC contact person
+- Czech language version of the Code was not accessible on the website
```

Comment:
Nadpis se opakuje, v odstavcích chybí tečky na konci vět.
Summary... was not posted at the venue.

Reply:
Podnadpis, takže se vztahuje k jinému tématu, než ten předchozí.